app/commands/incident.py

```diff
@@ -306,9 +306,12 @@ def submit(ack, view, say, body, client, logger):
     text = ":alphabet-yellow-question: Is someone `penetration or performance testing`? Please stop it to make your life easier."
     say(text=text, channel=channel_id)
 
+
     # Invite oncall to channel
     for user in oncall:
-        client.conversations_invite(channel=channel_id, users=user["id"])
+        # if the incident creator is also oncall, don't invite them again
+        if user is not user_id: 
```

Comment:
So I realized that I added the same bug a few lines down where I invite in the security group users.  What would you think of merging these 2 invite block so that we only did it once and only need to make the `if user is not user_id` check once?

Reply:
Sure! I can certainly merge those 2 blocks. I actually thought about this as well when I was fixing this bug and was going to run it by you.

Reply:
@patheard - I could not combine a single if statement because I needed to traverse both arrays (ie on_call and security_group) but I hope that this works. I tested it and it works. I can always change it if you think it is not satisfactory. Thanks!

Reply:
Totally fine by my the way you've done it!  I like how you're now only calling the `client.conversations_invite` method once now and building up the list of people to invite.